nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/rest/RestESDocumentsTest.java

```diff
@@ -140,6 +159,35 @@ public void isQueryEndpointCanSwitchToES() {
         }
     }
 
+    @Test
+    @Deploy("org.nuxeo.ecm.platform.collections.core:OSGI-INF/collection-pageprovider-contrib.xml")
+    public void iCanUseFulltextOperatorWithElasticsearchPageProvider() {
+        DocumentModel coll1 = session.createDocumentModel(ROOT_PATH, COLLECTION_NAME + 1, COLLECTION_TYPE);
+        coll1.setPropertyValue(DUBLINCORE_TITLE_PROPERTY, coll1.getName());
+        DocumentModel coll2 = session.createDocumentModel(ROOT_PATH, COLLECTION_NAME + 2, COLLECTION_TYPE);
+        coll2.setPropertyValue(DUBLINCORE_TITLE_PROPERTY, coll2.getName());
+        DocumentModel fufu = session.createDocumentModel(ROOT_PATH, "furtiveCollection", COLLECTION_TYPE);
+        fufu.setPropertyValue(DUBLINCORE_TITLE_PROPERTY, fufu.getName());
+
+        session.createDocument(coll1);
+        session.createDocument(coll2);
+        session.createDocument(fufu);
+
+        txFeature.nextTransaction();
+
+        PageProviderDefinition ppdef = pageProviderService.getPageProviderDefinition(COLLECTION_PAGE_PROVIDER);
+
+        Map<String, Serializable> props = Map.of(ElasticSearchNativePageProvider.CORE_SESSION_PROPERTY,
+                (Serializable) session);
+        PageProvider<?> pp = pageProviderService.getPageProvider(ppdef.getName(), ppdef, null, null, null, 0L, props,
```

Comment:
```suggestion
        PageProvider<DocumentModel> pp = (PageProvider<DocumentModel>) pageProviderService.getPageProvider(ppdef.getName(), ppdef, null, null, null, 0L, props,
```
Doing the cast here simplifies the following code.

Reply:
Done with a local `@SuppressWarnings("unchecked")`